src/routes/storage/revoke_file.ts

```diff
@@ -0,0 +1,62 @@
+import { Request, Response } from 'express'
+
+import Boom from '@hapi/boom'
+import { asyncWrapper } from '@shared/helpers'
+import { s3 } from '@shared/s3'
+import { v4 as uuidv4 } from 'uuid'
+import { S3_BUCKET, HASURA_GRAPHQL_ADMIN_SECRET } from '@shared/config'
+
+async function revokeFile(req: Request, res: Response): Promise<unknown> {
+  const key = req.params[0]
+
+  if (req.headers['x-hasura-admin-secret'] !== HASURA_GRAPHQL_ADMIN_SECRET) {
+    throw Boom.forbidden()
+  }
```

Comment:
Not sure if we should use the hasura graphql admin secret here or some other admins secret.